docs/source/redhat-opens-install.rst

```diff
@@ -197,10 +200,26 @@ On a control node, perform the following steps:
            ETCD_LISTEN_CLIENT_URLS="http://0.0.0.0:2379,http://0.0.0.0:4001"
            ETCD_LISTEN_PEER_URLS="http://0.0.0.0:2380"
            ETCD_INITIAL_ADVERTISE_PEER_URLS="http://<public_ip>:2380"
-           ETCD_INITIAL_CLUSTER_TOKEN="<cluster_id>"
            ETCD_INITIAL_CLUSTER="<hostname>=http://<public_ip>:2380"
            ETCD_INITIAL_CLUSTER_STATE=new
 
+     For RHEL 7, Check the ``uuidgen`` tool is installed (the output should
+     change each time)::
```

Comment:
@alexwlchan Can you confirm that 7cb302016cdf0261bcbe703e762dde686977d737 resolves this problem?